### DISCUSSION TITLE
[2022/05/17] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#205] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/204 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by kobayang** [comment]
https://coliss.com/articles/build-websites/operation/work/material-design-3-whats-new.html

**Comment by kobayang** [comment]
https://riffle.systems/essays/prelude/

**Comment by kobayang** [comment]
https://docs.google.com/presentation/d/e/2PACX-1vSjF1cp8fAdMKwFkLmtFT66e-UCD6zanO5KLsuRFmWyZ-8c06YQmgWT750Aqai2JfqHRARYWlkDLfaB/pub?slide=id.p
